README.md

```diff
@@ -224,6 +224,17 @@ If [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy), the modul
 
 If a value of `'minimal'` is set, the progress indicator will render as a small, colored bar at the top of the window. This can be useful when the default fancy progress indicator interferes with elements in the page.
 
+### `ramdisk`
+Type: `boolean`<br>
```

Comment:
I'd like to request this to also be an object, that would fit the options object:
https://github.com/shellscape/webpack-plugin-ramdisk#options

Reply:
Yeah, better with an object. 👍

Reply:
Under what scenario(s) is the average user going to need to increase either the `bytes` or the `blockSize` options?

Reply:
@shellscape I personally don't, but these are the options the ramdisk plugin has. I'm sure the defaults are good. But you choose to make them configurable for a reason?

I'm just advocating for passing through any option to the ramdisk plugin, without hardcoding in the plugin-serve.

Reply:
I see. I like to take a stepped, metered approach for webpack-related things. e.g. when the need arises and a request is made to open those options up, we'll modify the `ramdisk` option to allow for an `Object`. I also don't feel that it will require a `vote:...` issue since it's pretty straightforward. Otherwise, that's a branch of code that needs testing and maintenance that we're not really sure whether people are using or not. We took a similar approach with the other options that allow for passing different types.